frontend/src/app/main/ui/workspace/sidebar/options/menus/typography.cljs

```diff
@@ -374,9 +374,14 @@
         line-height (or line-height "1.2")
         letter-spacing (or letter-spacing "0")
 
+        line-height-nillable (if (= (str line-height) "1.2") false true)
+
         handle-change
         (fn [value attr]
-          (on-change {attr (str value)}))]
+          (let [new-values (case attr
+                            :line-height (assoc values :line-height (if (or (str/empty? value) (nil? value)) 1.2 value))
```

Comment:
I think you can let the handle-change function as the original one if just add
```
:default "1.2"
```
To 
```
[:> numeric-input" ...
```

Reply:
Thank you so much for the suggestion. This worked like a charm without tinkering the handle-change function. I have made the changes